pkg/test/framework/components/echo/kube/deployment.go

```diff
@@ -58,64 +57,64 @@ spec:
   selector:
     app: {{ .Service }}
 ---
+{{$deployConfig := .Subsets }}
```

Comment:
nit: maybe cleanup the variable naming a bit?  Suggest:

`deployConfig`->`subsets`
`$w`->`subset`

Reply:
done.